src/webgpu/util/constants.ts

```diff
@@ -403,10 +413,17 @@ export const kValue = {
         sixth: reinterpretU32AsF32(kBit.f32.positive.pi.sixth),
       },
       e: reinterpretU32AsF32(kBit.f32.positive.e),
-      first_f64_not_castable: reinterpretU32AsF32(kBit.f32.positive.max) / 2 + 2 ** 127, // mid point of 2**128 and largest f32
-      last_f64_castable: reinterpretU64AsF64(
-        BigInt(reinterpretU32AsF32(kBit.f32.positive.max) / 2 + 2 ** 127) - BigInt(1)
```

Comment:
The just raw converting this to a BigInt I think was wrong, since this is getting passed through a bitfield -> float conversion. I am pretty sure there needed to be a float -> bitfield conversion, to 1 bit could be removed, before converting to a float... I did the same thing for the other instances of this